### ISSUE TITLE
Feedback (No actions)

### ISSUE THREAD

**Issue by bwarren2** [aws/copilot-cli#1472]
TLDR: I LOVE YOUR WORK!

I have been wanting to get into container-oriented service architectures for years, but every time I tried ECS for something serious I bounced off: I hit some snag, or some unintuitive CLI response, or the recommended pattern was another AWS product that [1] recommended another AWS product that GOTO 1,  and eventually gave up.  Fargate was a big help, and getting things done in the console was OKish, but I am still not super confident.

Last night I was exploring and stumbled upon your project.  People can usually pick up professionalism really quickly based on the little details, and your project instantly stood out.  When I actually _used_ it, I was even more impressed.  

Some little things are definitely me doing something wrong, or hypothetically a bug:  When I stood up a basic Django project, the "view logs" link in the cluster was broken, but in actual Cloudwatch there was a different log group that was correct.  I wish I knew better how to diagnose 503 errors than blind-guessing that the service was in PENDING but the health check was failing.

The single biggest thing I would like to hear more about is how to use `copilot` as a base, and augment while y'all continue building wonderful features.  Is it reasonable to set a copilot service and spawn an RDS instance into that VPC, while you set up adding RDS storages?  How about adding autoscaling rules?

I would **love** to set up a project "the real way" using `copilot`, write it up, and share it more broadly to get people to use the project.  I took casual notes for me on my still-gestating [blog](https://trusting-kowalevski-78744f.netlify.app/posts/fargate/).  Are any of you available for questions/pairing on what a blessed setup/approach is?  What is your hourly consulting rate?

Thank you so much for your excellent work.  You have a fan in me.  Keep it up!

**Comment by efekarakus** [comment]
❤️ Thank you so much Ben! This type of feedback is so motivating and we really appreciate it :)

>  I wish I knew better how to diagnose 503 errors than blind-guessing that the service was in PENDING but the health check was failing.

We have some plans here related to https://github.com/aws/copilot-cli/issues/1421 where while the service is deploying we'd like to stream logs from your service to the terminal so you can see what is going on, as well as seeing if any tasks are failing to come up. I think that will help debug healthcheck failures.

>  Is it reasonable to set a copilot service and spawn an RDS instance into that VPC, while you set up adding RDS storages? How about adding autoscaling rules?

Yeah! We're currently working on making our docs more user friendly :) We're writing a little application at the moment to show case what a typical copilot application should look like: https://github.com/copilot-example-voting-app, but in the mean time if you're curious about using RDS the API service might be interesting for you: https://github.com/copilot-example-voting-app/api

We have an  [Additional AWS Resources](https://github.com/aws/copilot-cli/wiki/Additional-AWS-Resources) feature to integrate with other AWS services. And in the API service you can find the database here: https://github.com/copilot-example-voting-app/api/tree/main/copilot/api/addons

> How about adding autoscaling rules?

Autoscaling is supported within the count field (https://github.com/aws/copilot-cli/wiki/Load-Balanced-Web-Service-Manifest)
```yaml
count:
  range: 1-10
  cpu_percentage: 70
  memory_percentage: 80
```

> Are any of you available for questions/pairing on what a blessed setup/approach is?

Feel free to create new issues or just comment here and one of us will help answering questions, this way it would be discoverable for everyone using Copilot :)
